docs/data/stashcache/run-stashcache-container.md

```diff
@@ -53,18 +55,54 @@ Further behavior of the cache can be configured by setting the following in the
 Running a Cache
 ---------------
 
-To run the container, use `docker run` with the following options, replacing the text within angle brackets
-with your own values:
+StashCache cache containers  may be run with either multiple mounted host partitions (recommended) or a single host
+partition.
+
+!!!important "Partition ownership"
+    Host partitions mounted into the StashCache origin container must be owned `10940:10940`
```

Comment:
I thought you fixed this by code like the cache container does?

Reply:
Good call! I forgot about that